docs/configure/user-interface/features-and-behavior.mdx

```diff
@@ -39,9 +39,20 @@ The following options are configurable under the `sidebar` namespace:
 
 The following options are configurable under the `toolbar` namespace:
 
-| Name   | Type   | Description                        | Example Value       |
-| ------ | ------ | ---------------------------------- | ------------------- |
-| **id** | String | Toggle visibility for toolbar item | `{ hidden: false }` |
+| Name     | Type   | Description                                                          | Example Value        |
+| ---------| ------ | -------------------------------------------------------------------- | -------------------- |
+| **[id]** | String | Toggle visibility for a specific toolbar item (e.g. `title`, `zoom`) | `{ hidden: false }`  |
+
+## TODO layout customisation doc
+
+TODO:
+1. Document both functions with use cases
+2. Add caveat about bricking the user experience by hiding sidebar everywhere
```

Comment:
@Sidnioulz, could you ping me when you finish adding the documentation so that I can review it properly and help you get this in? 

Let me know and we'll go from there.

Reply:
@jonniebigodes thanks, I had totally forgotten about this bit. I've updated it to integrate the new config options in your existing docs, and added a `Sidebar UI` section with both the showSidebar options/API and the keyboard shortcut doc which was missing.

I've placed a warning about not using `showSidebar` without an alternative means of nav everywhere where it's mentioned.